src/Microsoft.AspNetCore.Dispatcher/Microsoft.AspNetCore.Dispatcher.csproj

```diff
@@ -12,6 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" />
```

Comment:
This should be Microsoft.AspNetCore.Hosting.Abstractions

Reply:
But I want to use `IStartupFilter` which is in this namespace https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Hosting.Abstractions/IStartupFilter.cs#L7